### PR TITLE
Add t75 keyboard

### PR DESCRIPTION
**Pull request by JackyJia73**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* New keyboard t75

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/vertex/t75/info.json [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    
+    "diode_direction": "ROW2COL",
+
+    "matrix_pins": {
+        "cols": ["B0", "B2", "B10", "B11", "A15", "B3","B4", "B5",  "B6", "B7", "B8", "B9", "B12", "B13","B14"],
+        "rows": ["A0", "A7", "A6", "A5", "A4","A3"]
+    },
+    "indicators": {
+        "caps_lock": "B1",
+        "on_state": 1
```
Comment (fauxpark) [inline keyboards/vertex/t75/info.json:34]:
```suggestion
        "caps_lock": "B1"
```

File: keyboards/vertex/t75/info.json
Comment (fauxpark) [inline keyboards/vertex/t75/info.json:1]: Please run `qmk format-json -i` on this file.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/vertex/t75/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+	[0] = LAYOUT(
+		KC_ESC,           KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_PSCR, KC_PSCR, KC_PAUS, 
+		KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_INS,  KC_HOME, KC_PGUP,
+		KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL,  KC_END,  KC_PGDN,
+		KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,    KC_ENT,  
+		KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,             KC_RSFT,                KC_UP,
+		KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT,   MO(1), KC_APP, KC_RCTL, KC_LEFT, KC_DOWN, KC_RIGHT,
+		_______, _______
+	)
```
Comment (fauxpark) [inline keyboards/vertex/t75/keymaps/default/keymap.c:30]:
```suggestion
	),
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/vertex/t75/info.json
Comment (JackyJia73) [reply keyboards/vertex/t75/info.json:1]: Done

Comment (fauxpark) [reply keyboards/vertex/t75/info.json:1]: Doesn't seem like it.

Comment (JackyJia73) [reply keyboards/vertex/t75/info.json:1]: Something wrong : You have not concluded your merge.

File: keyboards/vertex/t75/info.json [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": false,
+        "rgblight": false
```
Comment (waffle87) [inline keyboards/vertex/t75/info.json:15]:
```suggestion
        "extrakey": true,
        "mousekey": true,
        "nkro": true
```

File: keyboards/vertex/t75/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+		KC_LCTL, KC_LALT, KC_LGUI,                            KC_SPC,                             KC_RALT,   MO(1), KC_LEFT, KC_DOWN, KC_RIGHT
+	),
+	[3] = LAYOUT(
+       	_______,  KC_BRID,   KC_BRIU,   KC_MCTL,   KC_LPAD,   _______,   _______,   KC_MRWD,   KC_MPLY,   KC_MNXT,   KC_MUTE,  KC_VOLD,  KC_VOLU,  _______,
+		_______,  _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,  _______,  _______,  _______, _______,
+		_______,  _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,  _______,  _______,  _______, _______, 
+		_______,  _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,  _______,  _______,  _______,
+		_______,  _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,  _______,  _______,  _______,
+		_______,  _______,   _______,                         _______,                         _______,   _______,   _______,  _______,  _______
```
Comment (waffle87) [inline keyboards/vertex/t75/keymaps/default/keymap.c:56]: Neither of these layers are accessible.
```suggestion
```

File: keyboards/vertex/t75/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+		KC_LCTL, KC_LALT, KC_LGUI,                            KC_SPC,                             KC_RALT,   MO(1), KC_LEFT, KC_DOWN, KC_RIGHT
+	),
+	[3] = LAYOUT(
+       	_______,  KC_BRID,   KC_BRIU,   KC_MCTL,   KC_LPAD,   _______,   _______,   KC_MRWD,   KC_MPLY,   KC_MNXT,   KC_MUTE,  KC_VOLD,  KC_VOLU,  _______,
+		_______,  _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,  _______,  _______,  _______, _______,
+		_______,  _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,  _______,  _______,  _______, _______, 
+		_______,  _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,  _______,  _______,  _______,
+		_______,  _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,  _______,  _______,  _______,
+		_______,  _______,   _______,                         _______,                         _______,   _______,   _______,  _______,  _______
```
Comment (waffle87) [inline keyboards/vertex/t75/keymaps/via/keymap.c:56]:
```suggestion
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Review by waffle87 — APPROVED** [review]

File: keyboards/vertex/t75/info.json
Comment (JackyJia73) [reply keyboards/vertex/t75/info.json:1]: Done

File: keyboards/vertex/t75/keymaps/default/keymap.c (hunk 2 above)
Comment (JackyJia73) [reply keyboards/vertex/t75/keymaps/default/keymap.c]: updated

**Review by fauxpark — APPROVED** [review]

**Pull request merged**